cmd/gateway-main.go

```diff
@@ -63,7 +63,8 @@ EXAMPLES:
       $ {{.HelpName}} s3 https://play.minio.io:9000
 
   4. Start minio gateway server for GCS backend.
-      $ gcloud init --console-only # this will configure your google account
+      $ export GOOGLE_APPLICATION_CREDENTIALS=/path/to/credentials.json
```

Comment:
What a user needs to do for Minio GCS gateway to successfully authenticate with the GCS account depends on the environment it is being run. This message applies only when the gateway is run _outside_ GCP. Is it possible for the gateway to know if it's running in a GCP instance? If not, we could modify the message to,

> (When run outside Google Cloud Platform)
$ export GOOGLE_APPLICATION_CREDENTIALS=/path/to/credentials.json